apps/ledger-live-desktop/src/renderer/components/FirebaseRemoteConfig.tsx

```diff
@@ -14,6 +15,10 @@ export const useFirebaseRemoteConfig = () => useContext(FirebaseRemoteConfigCont
 
 export const formatToFirebaseFeatureId = (id: string) => `feature_${snakeCase(id)}`;
 
+export const formatCurrencyIdToFeatureKey = (id: string) => {
+  return `currency${startCase(id).replace(/\s/g, "")}`;
```

Comment:
Just to understand, nothing mandatory, why do we use startCase and not SnakeCase ?

Reply:
Yeah would be better to use snake_case everywhere but remotely the feature flags are snake_case and locally they are in camelCase

So in remote it's `feature_currency_avalanche_c_chain` and in the code it's `currencyAvalancheCChain`